### PR TITLE
(QENG-144) created masterless option, and support for masterless installs

### PR DESCRIPTION
**Pull request by kevpl**

### CODE REVIEW

**Comment by anodelman** [comment]
@kevpl I'll assume that this passed local masterless installation testing?

**Comment by kevpl** [comment]
@anodelman yes, it has

File: lib/beaker/dsl/install_utils.rb [hunk 1]
```diff
@@ -453,12 +453,16 @@ def deploy_frictionless_to_master(host)
       # @api private
       #
       def do_install hosts, opts = {}
```
Comment (anodelman) [inline lib/beaker/dsl/install_utils.rb]: Yard docs needs to be updated here to describe the new option.

Comment (kevpl) [inline lib/beaker/dsl/install_utils.rb]: @anodelman `do_install` actually doesn't take the masterless option, it overwrites it from the main options hash.  This happens a few lines below this comment.

**Pull request merged**